rest-service/rest-service-B/src/main/java/com/ivoyant/rest_service_B/controller/ServiceBController.java

```diff
@@ -0,0 +1,58 @@
+package com.ivoyant.rest_service_B.controller;
+
+import com.ivoyant.rest_service_B.service.StudentService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@Slf4j
+public class ServiceBController {
+
+    private final StudentService studentService;
+
+    @Autowired
+    public ServiceBController(StudentService studentService) {
+        this.studentService = studentService;
+    }
+
+    @GetMapping("/service-b/api/resource")
+    public ResponseEntity<List<Student>> getResource() {
+        List<Student> allData = studentService.getAllStudents();
```

Comment:
Try adding exception handler and also return the negative response.

Reply:
updated